apps/metadata-editor/src/app/dashboard/sidebar/sidebar.component.ts

```diff
@@ -41,4 +45,21 @@ export class SidebarComponent implements OnInit {
       (orgs, me) => orgs.filter((org) => org.name === me?.organisation)
     )
   }
+
+  logOut() {
+    const current_url = window.location.toString()
```

Comment:
Would it make sense to use `window.location.hostname` here?
I'm wondering how this works, since `window.location` returns an object, but I tested it and it returns the href.
I would rather use the hostname here, since you don't want to redirect to e.g. /edit/id12345.

Reply:
Or maybe even better to use the login url?
Since when I'm logged out, I want to see the login page again, right?

Reply:
Not sure we can call `LOGIN_URL`, I think it becomes `loginUrl` when exported from the service & therefore uses the `window.location`. I think using the hostname is easier too, will do that.